### PR TITLE
generated yeomen file for deamon in darwin and linux env. Only works on python3 

### PR DESCRIPTION
**Pull request by stran-c-cs**

### CODE REVIEW

**Comment by stran-c-cs** [comment]
@dwisecup-cs 
Can't add reviewers

File: generators/skill/templates/daemon/scripts/common/darwin/build-daemon.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+echo "Building Source"
+mkdir $SCRIPT_DIR/build
+cp -r $SCRIPT_DIR/src/ $SCRIPT_DIR/build
+
+echo "Building Docker Image from $SCRIPT_DIR/build"
+docker build   -t "$IMAGE_NAME" "$SCRIPT_DIR"
+
+echo "Tagging with $PRIVATE_REGISTRY/$IMAGE_NAME:"$VERSION""
+docker tag $IMAGE_NAME $PRIVATE_REGISTRY/$IMAGE_NAME:"$VERSION"
```
Comment (dwisecup-cs) [inline generators/skill/templates/daemon/scripts/common/darwin/build-daemon.sh]: `docker tag $IMAGE_NAME "$PRIVATE_REGISTRY/$IMAGE_NAME:$VERSION"`

File: generators/skill/templates/daemon/scripts/common/darwin/deploy-daemon.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+IMAGE_NAME="<%= projectPrefix %><%= skillName %>"
+PRIVATE_REGISTRY=<%= privateRegistry %>
+PORT=5000
+
+cortex docker login --profile <%= profile %>
+echo "Running the build"
+${SCRIPT_DIR}/build-daemon.sh
+echo "Tagging the image"
+docker tag $IMAGE_NAME $PRIVATE_REGISTRY/$IMAGE_NAME:"$VERSION"
```
Comment (dwisecup-cs) [inline generators/skill/templates/daemon/scripts/common/darwin/deploy-daemon.sh]: `docker tag $IMAGE_NAME "$PRIVATE_REGISTRY/$IMAGE_NAME:$VERSION"`

File: generators/skill/templates/daemon/scripts/common/darwin/deploy-daemon.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+echo "Pushing Repo Changes"
+docker push  $PRIVATE_REGISTRY/$IMAGE_NAME:"$VERSION"
```
Comment (dwisecup-cs) [inline generators/skill/templates/daemon/scripts/common/darwin/deploy-daemon.sh]: ditto

File: generators/skill/templates/daemon/scripts/common/linux/deploy-daemon.sh [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+IMAGE_NAME="<%= projectPrefix %><%= skillName %>"
+PRIVATE_REGISTRY=<%= privateRegistry %>
+PORT=9091
+
+cortex docker login --profile <%= profile %>
+echo "Running the build"
+${SCRIPT_DIR}/build-daemon.sh
+echo "Tagging the image"
+docker tag $IMAGE_NAME $PRIVATE_REGISTRY/$IMAGE_NAME:$VERSION
```
Comment (dwisecup-cs) [inline generators/skill/templates/daemon/scripts/common/linux/deploy-daemon.sh]: `docker tag $IMAGE_NAME "$PRIVATE_REGISTRY/$IMAGE_NAME:$VERSION"`

File: generators/skill/templates/daemon/scripts/common/linux/deploy-daemon.sh [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+echo "Pushing Repo Changes"
+docker push  $PRIVATE_REGISTRY/$IMAGE_NAME:$VERSION
```
Comment (dwisecup-cs) [inline generators/skill/templates/daemon/scripts/common/linux/deploy-daemon.sh]: ditto

File: generators/skill/templates/daemon/scripts/node8/darwin/build-function.sh [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+set -e
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [ -d $SCRIPT_DIR/build ]; then
+    rm -rf $SCRIPT_DIR/build
+fi
+mkdir -p $SCRIPT_DIR/build/stage
+
+# copy source to a staging folder
+cp -r $SCRIPT_DIR/package.json $SCRIPT_DIR/src/index.js $SCRIPT_DIR/build/stage
```
Comment (dwisecup-cs) [inline generators/skill/templates/daemon/scripts/node8/darwin/build-function.sh]: Quote these too

File: generators/skill/templates/daemon/template/common/skill.yaml [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+    title: input1
+    parameters:
+      - name: text
+        type: string
+        description: Update this description
+        required: true
+    routing:
+      all:
+        action  : <%= projectPrefix %><%= functionName %>
+        runtime: cortex/functions
```
Comment (dwisecup-cs) [inline generators/skill/templates/daemon/template/common/skill.yaml]: cortex/daemons

File: generators/skill/templates/daemon/template/python2/src/__main__.py [hunk 8]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (dwisecup-cs) [inline generators/skill/templates/daemon/template/python2/src/__main__.py]: What is this function doing?

Comment (stran-c-cs) [reply generators/skill/templates/daemon/template/python2/src/__main__.py]: It's an extra space, doesn't affect anything

**Pull request merged**